cves/CVE-2017-12615.yml

```diff
@@ -204,4 +218,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
Was there any reference to a requirement? Were there any automated tests for this section of code?